### ISSUE TITLE
[HL2DM] Disabling Footsteps doesn't work

### ISSUE THREAD

**Issue by trk305** [ValveSoftware/Source-1-Games#6790]
Disabling Footsteps for your SERVER doesn't work (broken after the 20th anniversary probably)

Video: (can't upload it to github due to file size)

[https://www.youtube.com/watch?v=BtUUqVCQIYI](Video)

**Comment by adelina1985a** [comment]
@trk305 Yeah, we touched on this topic in a different issue recently. That's because `mp_footsteps 0` doesn't turn them off ~~(anymore)~~. Try the new `sv_footsteps 0` instead, until this is resolved.

In the Create Game->Server tab, the Footsteps option only changes mp_footsteps, so the ways to disable footsteps now are:
- add the new command in your 'listenserver.cfg' file before creating the game; or
- put the new command in the console after creating the game, every time you create a new game; or
- use a plugin or custom code; or
- make your own 'settings_default.scr' file! by duplicating the settings.scr and adding an entry for sv_footsteps there. I say adding and not replacing because the settings.scr will still generate that entry.

```
	"sv_footsteps"
	{
		"Real Footsteps"
		{ BOOL }
		{ "0" }
	}

	"mp_footsteps"
	{
		"Fake Footsteps"
		{ BOOL }
		{ "1" }
	}
```
![image](https://github.com/user-attachments/assets/675a4536-55eb-44f8-ba1d-bfbe26fa1073)

EDIT: Scratched out 'anymore', as the mp cvar didn't work before the update, per Xeogin. Tested and confirmed.

**Comment by trk305** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/6790#issuecomment-2508899576

Thank you very much.

**Comment by Xeogin** [comment]
Informational: mp_footsteps didn't do anything before the anniversary update either

**Comment by RiverHornet3209** [comment]
still not fixed after feb 18 2025 update

**Comment by RiverHornet3209** [comment]
Informational: `mp_footsteps` also doesn't do anything in HLDM:S, CS:S, DOD:S and TF2.
